docs/changelog.rst

```diff
@@ -7,8 +7,23 @@ Change log
 Current version
 ===============
 
+Additions
+---------
+
+* Added testing for Python 3.9
+
+Changes
+-------
+
+* Removed versioneer
```

Comment:
If you remove Python support then you also need to 
- mention it changelog
- bump the version (check what semver requires if you're following it)